packages/admin-ui/src/pages/rollups/components/useZkEvmConfig.ts

```diff
@@ -0,0 +1,31 @@
+import { useState, useEffect } from "react";
+import { ZKEVMItem } from "../../../../../types/src/rollups";
```

Comment:
this item outside the module, you should import it as follows `from "@dappnode/types"`